app.js

```diff
@@ -60,6 +62,26 @@ app.use(paths.health, healthcheck.configure({
     }
 }));
 
+app.get(paths.landingPages.overview, (req, res) => {
+   res.render('overview/template.html', overviewContent);
+});
+
+app.get(paths.landingPages.beforeYouAppeal, (req, res) => {
+    res.render('before-you-appeal/template.html');
+});
+
+app.get(paths.landingPages.helpWithAppeal, (req, res) => {
+    res.render('help-with-appeal/template.html');
+});
+
+app.get(paths.landingPages.startAnAppeal, (req, res) => {
+    res.render('start-an-appeal/template.html');
+});
+
+app.get(paths.landingPages.afterYouAppeal, (req, res) => {
+    res.render('after-you-appeal/template.html');
+});
+
```

Comment:
Can you move all of this out of app.js and add it to a separate file so we keep app.js nice and light. As there are other PRs outstanding feel free to make these minor changes when it's suitable to do so.